README.md

```diff
@@ -47,3 +55,4 @@ See the [docs] for more information on how to use `kpt`.
 [windows]: https://storage.googleapis.com/kpt-dev/latest/windows_amd64/kpt.exe
 [docs]: https://googlecontainertools.github.io/kpt
 [release]: https://github.com/GoogleContainerTools/kpt/releases/latest
+[gcr.io/kpt-dev/kpt]: https://pantheon.corp.google.com/gcr/images/kpt-dev/GLOBAL/kpt/
```

Comment:
Can we use a link there that doesn't refer to internal google names?

Reply:
Nice catch, updated link in both README and installation page on the site.